### PR TITLE
Test implemented for execute_move_effect() function

### PR DESCRIPTION
**Pull request by anchristensen**
I added a test for the execute_move_effect function checking whether it works when it is given a pre-existing list of moves. I also renamed the test implemented by @nbilaver so it was clear the difference between my test (uses existing list of moves to test function) and his (uses empty list to test function). This test is to finish the issue "Finish execute_move_effect() implementation" #1152

### CODE REVIEW

**Review by nbilaver — APPROVED** [review]
Everything looks good here!

**Comment by jacknugent1529** [comment]
@nbilaver and @anchristensen, you both worked on this, right? If so, the reviewer should be someone on your team who didn't work on the issue. Also, can you please add the appropriate assignees, labels, and milestones. Lastly, it would be useful to mention what issue(s) this is related to

**Review by arya32hub — COMMENTED** [review]
Created framework for execute_move_effect. Waiting on RPG battles to finish

**Review by arya32hub — APPROVED** [review]

**Review by jacknugent1529 — COMMENTED** [review]
Nice work! Just add a comment explaining what the test does and then you should be good

File: tests/skilltrees/effect_test.c [hunk 1]
```diff
@@ -27,6 +27,48 @@ Test(effect_tests, make_move_effect_test)
 
 }
 
+Test(effect_tests, execute_move_effect_test_empty_list)
```
Comment (jacknugent1529) [inline tests/skilltrees/effect_test.c:34]: Add a comment before the test explaining what it is testing for. See the other tests for examples

Comment (anchristensen) [reply tests/skilltrees/effect_test.c:34]: I just added comments explaining the test and pushed it, please let me know if there's anything else to fix!

**Review by jacknugent1529 — APPROVED** [review]
Looks good to me!

**Comment by jacknugent1529** [comment]
PR Score: E

Comments: Very nice work on your first PR! The code is well-organized and thoroughly tests the functionality.

**Pull request merged**